### PR TITLE
tests for shared logger

### PR DESCRIPTION
**Pull request by lkostrowski**
## Checklist

- [ ] I added changesets and [read good practices](/.changeset/README.md).

### CODE REVIEW

File: packages/logger/src/logger.test.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        childScopePrimitiveArg: 2,
+        childScopeObjectArg: {
+          objectKey: "objectValue",
+        },
+      });
+
+      /**
+       * TODO this assertion fails, but strings are the same. Why?
+       */
+      expect(console.log).toBeCalledWith(" 2024-02-01T04:15:00.000Z :Test Logger \tTest Message", {
```
Comment (lkostrowski) [inline packages/logger/src/logger.test.ts:33]: Can you try to run it locally? This assertion fails, but strings are equal.

Comment (krzysztofzuraw) [reply packages/logger/src/logger.test.ts:33]: I've played a bit with this an apparently this string will work:

```
[2m 2024-02-01T04:15:00.000Z :Test Logger[0m 	Test Message
```

I think we can also use [toMatchSnapshot()](https://vitest.dev/api/expect#tomatchsnapshot) instead 🤔

Comment (Cloud11PL) [reply packages/logger/src/logger.test.ts:33]: `"\x1B[2m 2024-02-01T04:15:00.000Z :Test Logger\x1B[0m \tTest Message"` worked for me

Comment (lkostrowski) [reply packages/logger/src/logger.test.ts:33]: Thanks, snapshot works too. I updated this, forgot about color characters

File: packages/logger/src/logger.test.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+      logger.info("Test Message", {
+        childScopePrimitiveArg: 2,
+        childScopeObjectArg: {
+          objectKey: "objectValue",
+        },
+      });
+
+      /**
+       * TODO this assertion fails, but strings are the same. Why?
```
Comment (krzysztofzuraw) [inline packages/logger/src/logger.test.ts:31]: **Suggestion:** this TODO can be removed

Comment (lkostrowski) [reply packages/logger/src/logger.test.ts]: done

**Review by krzysztofzuraw — APPROVED** [review]

**Pull request merged**